tests/testthat/test_Spectra.R

```diff
@@ -1096,6 +1096,93 @@ test_that("filterRt,Spectra works", {
     expect_equal(rtime(res), rtime(sps))
 })
 
+test_that("filterRanges, Spectra works", {
+    filt_spectra <- filterRanges(sps_dia, spectraVariables = c("rtime",
+                                                               "precursorMz",
+                                                               "peaksCount"),
+                                 ranges = c(30, 350, 200,500, 350, 600))
+    # test does not accept logical
+    logical_test <- spectraVariables(sps_dia) %in% c("rtime", "precursorMz",
+                                                     "peaksCount")
+    expect_error(filterRanges(sps_dia, spectraVariables = logical_test,
+                              ranges = c(30, 350, 200,500, 350, 600)))
```

Comment:
please for `expect_error` always also check for the error message - that the one you expect is thrown. In this case simply do `expect_error(... , "character")` (i.e. use one of the words from the error message that is thrown).